docs/source/tutorials/upload.rst

```diff
@@ -0,0 +1,103 @@
+Submitting data
+---------------
+
+Please submit your electronic structures calculations for surface reactions! Data submissions will be part of the Surface Reactions app (essentially CatApp v2.0) and is open to all institutions. Furthermore, the atomic structures that are part of your reaction can be utilized for other apps.
+
+Your publication/dataset will be listed on the http://www.catalysis-hub.org/publications page, with a link to the publishers homepage (if already published). Your data will be easily accessible to other researchers, who will be able to browse through reaction energies and you atomic structures from your publication. 
+
+SUNCAT group members
+....................
+If you're a member of the SUNCAT group, you can add your data to one of the folders on the Sherlock or SUNCAT cluster: 
+
+* SHERLOCK : /home/winther/data_catapp/
+* SHERLOCK2 : /home/users/winther/data_catapp/
+* SUNCAT : /nfs/slac/g/suncatfs/data_catapp/
+
+On SHERLOCK2 you can go to the directory and run::
+
+  . CATKITENV/bin/activate
+
+to use the local installation of CatKit. Alternatively you can install your own version of CatKit - see instructions below.
+
```

Comment:
I would add comment like: 

You should see a `(CATKITENV) `at the beginning of your prompt now to indicate that all python script and libraries are first imported from the virtualenv. To return your shell to the previous state simply type `deactivate` or log out.